reference/7/Microsoft.PowerShell.Utility/ConvertFrom-Json.md

```diff
@@ -17,7 +17,7 @@ Converts a JSON-formatted string to a custom object or a hash table.
 ## SYNTAX
 
 ```
-ConvertFrom-Json [-InputObject] <String> [-AsHashtable] [-Depth <Int32>] [<CommonParameters>]
```

Comment:
[-NoEnumerate]

Reply:
@msin3789 Sorry I'm not sure what you mean with this comment?